en/docs/deploy/migrate/what-has-changed.md

```diff
@@ -657,4 +657,36 @@ In WSO2 Identity Server 6.0.0, the Log4j version is upgraded to ```Log4j2```. Th
 ``` js
 [hazelcast]
 hazelcast.logging.type = log4j2
+```
+### Change In Data Type for Boolean Attributes From Schemas Endpoint
+
+`scim2/Schemas` endpoint now returns the attribute type of boolean attributes as `boolean`. Before this fix, the attribute type was returned as `string`.
```

Comment:
Lets not use the word 'fix' in docs. Better to use `change` or `update` instead.

Reply:
`Before this fix` -> `In previous versions of the Identity Server`